__tests__/account/AccountCreateTransaction.test.ts

```diff
@@ -100,6 +102,11 @@ describe('AccountCreateTransaction', () => {
             "transactionvalidduration": {
                 "seconds": 120
             }
-        });
+        };
+        expect(txnBody).toStrictEqual(expectedTxBody);
+
+        expectedTx.body = expectedTxBody;
+
+        expect(expectedTx.toString()).toEqual(transaction.toString());
```

Comment:
`expectedTx.toString()` should be spitting out `[object Object]` in this context. I would check the string value itself.